scala/coding/README.md

```
@@ -0,0 +1,8 @@
+## sbt project compiled with Scala 3
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)